### PR TITLE
Component : Rating and Slider completed

### PR DESCRIPTION
**Pull request by yogesh-**
Rating and Slider component completed.

image

image

### CODE REVIEW

**Review by tushargoyal812 — COMMENTED** [review]
code looks clean.
file structure is good.
looks good to me

File: css/rating_slider.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.star_rating {
+  height: 8rem;
+  width: 20rem;
+  border: 0.2rem solid var(--main-primary-color);
+  margin: 2rem;
+}
+
+/* For Slider */
```
Comment (GeetaPogul) [inline css/rating_slider.css:12]: remove commented code

File: styles.css [hunk 2]
```diff
@@ -9,6 +9,7 @@
 @import url(./css/snackbar.css);
 @import url(./css/lists.css);
 @import url(./css/modal.css);
+@import url(./css/rating_slider.css);
 
 * {
   box-sizing: border-box;
```
Comment (GeetaPogul) [inline styles.css:15]: remove this extra places
use prettier formatter

**Review by GeetaPogul — APPROVED** [review]
looks good to me

File: documentation.html [hunk 3]
```diff
@@ -1508,6 +1587,29 @@ <h1>Made with lots of
     closeBtn.onclick = function () {
         modal.style.display = "none"
     }
+
+    // For Rating
+
+    const allstars = document.querySelectorAll(".star");
+
+    allstars.forEach((star, i) => {
+        star.addEventListener("click", () => {
```
Comment (Manishsharma203) [inline documentation.html:1596]: Code looks clean and good. Just a suggestion, here you can define the callback function separately and then call it here to keep the code more modular.

**Pull request merged**